ui/apps/dashboard/src/app/(organization-active)/(dashboard)/env/[environmentSlug]/manage/[ingestKeys]/CreateKeyButton.tsx

```diff
@@ -43,7 +44,11 @@ export default function CreateKeyButton() {
           name: inputValue,
           source: currentContent.type,
           metadata: {
-            transform: undefined,
+            // We must specify a transform, otherwise the webhook will be in a
+            // broken state due to a missing transform. It might be better to
+            // specify the default transform in the backend, but this is a quick
+            // fix
+            transform: defaultTransform,
```

Comment:
We should only set this when `currentContent.type` is `"webhook"`. We can't set this for event keys as within the Event API, the transform will execute even if it's an event key and not a webhook, but the user would never see this in the UI and it would lead to bugs: https://github.com/inngest/monorepo/blob/0fbc39c6e27d7fff41b7a8532df4495b2e7a0608/pkg/applogic/events/ingestapi/ingestapi.go#L181-L183

Additionally, to prevent a similar change like this from accidentally causing a but, we should probably also land a fix in monorepo to only execute a transform for webhook `source` keys only, not event keys.